src/Utilities/MoistThermodynamics/src/MoistThermodynamics.jl

```diff
@@ -33,6 +33,17 @@ export liquid_fraction, phase_partitioning_eq!, saturation_adjustment, phase_par
 # Auxiliary functions, e.g., for diagnostic purposes
 export liquid_ice_pottemp, dry_pottemp, exner
 
+
+"""
+    sound_speed(T,γ,R_gas)
+Returns the speed of sound in moist air.
+Arguments: Temperature, specific heat capacity ratio, moist gas constant 
+"""
+function sound_speed(T, γ, gas_constant_air)
+    return sqrt(T * γ * gas_constant_air) 
+end
+
+
```

Comment:
Compare with `soundspeed_air` in PR 137 and choose what makes more sense.

Reply:
I prefer #137 commit 8e3a992 with gas constants hidden; will fix accordingly.